src/index.html

```diff
@@ -5,9 +5,244 @@
+              <div class="testimonials__item-photo">
+                <img src="./images/people/2.png" alt="Leslie Mckinney">
+              </div>
+              <div class="testimonials__item-name">Scarlett Hawkins</div>
+              <div class="testimonials__item-prof">Senior Vice President</div>
+              <div class="testimonials__item-review">
+                My soul is illuminated by an <br>
+                unearthly joy, as these beautiful <br>
+                spring morning, which I enjoy <br>
+                with all my heart
+              </div>
+            </div>
+
+            <div class="testimonials__item">
+              <div class="testimonials__item-photo">
+                <img src="./images/people/3.png" alt="Leslie Mckinney">
+              </div>
+              <div class="testimonials__item-name">Max Henry</div>
+              <div class="testimonials__item-prof">Chief Design Officer</div>
+              <div class="testimonials__item-review">
+                When my lovely valley of steam <br>
+                rises and half-day sun is above an <br>
+                impermeable
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="main__section">
+        <div class="contact-us" id="contact-us">
+          <div class="contact-us__title">Contact us</div>
+          <div class="contact-us__items">
+            <div class="contact-us__items-1">
+              <div class="contact-us__items-1-phone">
+                <span class="contact-us__items-1-phone-title">Phone</span>
+                <span>+1 234 5555-55-55</span>
+              </div>
+              <div class="contact-us__items-1-email">
+                <span class="contact-us__items-1-email-title">Email</span>
+                <span>hello@miami.com</span>
+              </div>
+            </div>
+            <div class="contact-us__items-2">
+              <span class="contact-us__items-2-title">Address</span>
+              <span>
+                400 first ave. n.
+                <br>
+                suite 700
+                <br>
+                Minneapolis, MN
+                <br>
+                55401
+              </span>
+            </div>
+            <div class="contact-us__items-3">
+              <div class="contact-us__items-3-name">
+                <input
+                  class="contact-us__items-3-input"
+                  type="text"
+                  placeholder="Name"
+                  name="name"
+                  required
+                >
+              </div>
+
+              <div class="contact-us__items-3-email">
+                <input
+                  class="contact-us__items-3-input"
+                  type="email"
+                  placeholder="Email"
+                  name="email"
+                  required
+                >
+              </div>
+
+              <div class="contact-us__items-3-message">
+                <textarea
```

Comment:
```suggestion
                <textarea
```
Don't use these wrappers.
Try to make the classes for your inputs instead of creating multiple similar `div` wrappers